src/platform/forms-system/test/js/review/ReviewCollapsibleChapter.unit.spec.jsx

```diff
@@ -227,12 +227,7 @@ describe('<ReviewCollapsibleChapter>', () => {
       {
         pageKey: 'test1',
         title: '',
-        schema: {
-          properties: {
-            condition1: 'boolean',
-            condition2: 'boolean',
-          },
-        },
+        schema: {},
```

Comment:
What are these changes for?

Reply:
Before this PR, invalid schema properties in theses tests wasn't causing any issues, but since I'm expected schema properties to be valid I had to fix some issues.

But this is actually also just cleanup, since the schema on the page object isn't actually used in `ReviewCollapsibleChapter`